CHANGELOG.md

```diff
@@ -3,7 +3,6 @@ All notable changes to this project will be documented in this file.
 This project adheres to [Semantic Versioning](http://semver.org/) and [Keep a changelog](https://github.com/olivierlacan/keep-a-changelog).
 
 ## [Unreleased](https://github.com/idealista/fluentd-role/tree/develop)
-

```

Comment:
What if you add this change to the changelog? 🤗

Reply:
Looks like I deleted the line before commiting :thinking: . Anyway, added the change